docs/advanced_topics/settings.rst

```diff
@@ -338,20 +321,76 @@ These two files should reside in your project directory (``myproject/myproject/`
 
   import os
 
-  PROJECT_ROOT = os.path.join(os.path.dirname(__file__), '..', '..')
+  PROJECT_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+  BASE_DIR = os.path.dirname(PROJECT_DIR)
 
   DEBUG = True
-  TEMPLATE_DEBUG = DEBUG
 
-  ADMINS = (
-      # ('Your Name', 'your_email@example.com'),
-  )
+  # Application definition
 
-  MANAGERS = ADMINS
+  INSTALLED_APPS = [
```

Comment:
We list the installed apps in the opposite order in the project template: https://github.com/torchbox/wagtail/blob/bc8322bf7e91c1a2606392090e24cd78c9e5a817/wagtail/project_template/project_name/settings/base.py#L26-L52

see: #1536